source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -18,3 +18,19 @@ To change the timezone of a domain, go to Configuration > Regional settings. In
    :class: note
 
    It is also possible to directly write a city or zone in the textbox. A selection of matching existing timezones will appear.
+
+
+Datasets default location
+-------------------------
+
+The :ref:`geographic coverage metadata <geographic-coverage-metadata>` indicates the location of the data contained in the dataset. Although the metadata can be set individually for each dataset, it is possible to configure the default location(s) that will be automatically set as geographic coverage metadata for all datasets created afterward.
```

Comment:
indicate the "territory covered by the data contained" ?

Reply:
The meta is named "Geographic coverage", it may be a good idea to use its name to be explicit.